thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss

```diff
@@ -356,7 +356,7 @@ body {
 
   &__container {
     display: flex;
-    justify-content: flex-end;
+    justify-content: flex-start;
```

Comment:
any reason we need to specifically set this? default value is `flex-start`

Reply:
I'll remove this line then. Didn't realize `flex-start` is the default. Checked the UI, and it looks fine after removal.